src/main/java/com/the/dark/side/crew/fejsbuk/model/UserEntity.java

```diff
@@ -17,13 +17,18 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
 
-    @Column(name = "first_name")
     private String firstName;
 
-    @Column(name = "last_name")
     private String lastName;
 
     private String login;
 
     private String email;
+
+    public UserEntity(String firstName, String lastName, String login, String email) {
```

Comment:
If you're not using it, just remove and get rid of @NoArgsContractor. Same with postEntity